Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
+- [Azure_Subscription_Config_ASC_Setup_SecurityContacts](#Azure_Subscription_Config_ASC_Setup_SecurityContacts)
+- [Azure_Subscription_SI_No_Billing_Activity_Trial](#Azure_Subscription_SI_No_Billing_Activity_Trial)
+- [Azure_Subscription_Configure_Conditional_Access_for_PIM](#Azure_Subscription_Configure_Conditional_Access_for_PIM)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_Subscription_AuthZ_Remove_Deprecated_Accounts 
+
+### DisplayName 
+Remove Orphaned accounts from your subscription(s) 
+
+### Rationale 
+Deprecated accounts are ones that were once deployed to your subscription for some trial/pilot initiative (or some other purpose). These are not required any more and are a standing risk if present in any role on the subscription. 
+
+### Control Settings 
+```json 
+{
+    "DeprecatedAccounts": ""
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> No deprecated account is found at subscription scope (in both ASC and Reader scan).
+> 
+> **Failed:** 
+> Deprecated account is found at subscription scope (in any one of ASC and Reader scan).
+> 
+> **Verify:** 
+> ASC assessment status is not applicable or policy is missing.
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 Steps to remove role assignments of deprecated/invalid accounts are:  <br />a. To remove permanent role assignment use command 'Remove-AzRoleAssignment' or refer link, https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-remove#azure-portal <br />b. To remove classic role assignments, refer link: https://docs.microsoft.com/en-us/azure/role-based-access-control/classic-administrators#remove-a-co-administrator <br />c. To remove PIM role assignments, refer link https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-how-to-add-role-to-user?tabs=new#update-or-remove-an-existing-role-assignment. <br />For bulk remediation of permanent and classic role assignments using PowerShell, refer https://aka.ms/azts-docs/rscript/Azure_Subscription_AuthZ_Remove_Deprecated_Accounts.
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list role assignment at scope: - /{scope}/providers/Microsoft.Authorization/roleAssignments?api-version=2018-01-01-preview <br />
+**Properties:** principalId
+ <br />
+
+- PIM API to get role assignment: - /beta/privilegedAccess/azureResources/resources/{uniquePIMIdentifier}/roleAssignments?$expand=subject,roleDefinition($expand=resource)&$filter=(memberType ne '{filterCondition}') <br />
+**Properties:** subject.id
+ <br />
+
+- ARM API to list security assessments at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Security/assessments?api-version=2020-01-01<br />
+**Properties:** id, name, resourceDetails.Id, displayName, status.code, status, additionalData
+ <br />
+
+<br />
+
+___ 
+
+## Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities 
```

Comment:
Please check the list of APIs being used for this Control and the one following. They are the same except that the second Control focusses only on 'privileged' roles, unlike this Control which flags all roles. The APIs being used should ideally be the same.